integration/kafka/ReadMe.md

```diff
@@ -0,0 +1,107 @@
+##Kafka Alluxio Connect
+Kafka Alluxio Connector implements exporting kafka's specific topic data to alluxio filesystem.User can define data format stored in alluxio file,such as JsonFormat.
```

Comment:
Kafka Alluxio Connector exports data in Kafka to Alluxio. The connector supports user defined data format and shipped with JSON format.